sunpy/net/fido_factory.py

```diff
@@ -362,9 +362,10 @@ def fetch(self, *query_results, path=None, max_conn=5, progress=True,
         reslist = []
         for query_result in query_results:
             for block in query_result.responses:
-                reslist.append(block.client.fetch(block, path=path,
-                                                  downloader=downloader,
-                                                  wait=False, **kwargs))
+                if type(block.client).__name__ != 'HECClient':
```

Comment:
This is to prevent fido from fetching the hek results?

Reply:
Yes, it will ignore `HECClient`. If we make `hek` too compatible with Fido, Then we would just need to add an `and`.

Reply:
We need to come up with a better method.

Reply:
We can have a class memeber `self.ignore_fetch` which is False by default. For these clients, it can be set to True.

Reply:
Added `isMetaClient` as a bool class attribute in `BaseClient`